requirements.txt

```diff
@@ -25,3 +25,9 @@ PyYAML
 # Deps: observability_libs
 lightkube >= 0.11
 lightkube-models
+
+# Requests is an elegant and simple HTTP library for Python, built for human beings.
+# Code: https://github.com/psf/requests
+# Docs: https://requests.readthedocs.io/en/latest/
+# Deps: alertmanager_remote_configuration
+requests
```

Comment:
Can we avoid using `requests`?

- Is it a provider or a consumer dependency?
- See reference without using it: https://github.com/canonical/alertmanager-k8s-operator/blob/main/src/alertmanager_client.py

Reply:
Out of curiosity, why should we avoid using `requests`?

Reply:
It's fine to use `requests` but when usage is very simple and similar to stdlib's urllib, might as well reduce deps, which means avoid potential pinning fixes and reduce charm size.

Reply:
Done.